selfdrive/test/test_onroad.py

```diff
@@ -211,13 +211,13 @@ def test_cloudlog_size(self):
   def test_log_sizes(self):
     for f, sz in self.log_sizes.items():
       if f.name == "qcamera.ts":
-        assert 2.15 < sz < 2.6
+        assert 0.90 < sz < 2.6
```

Comment:
we've gotta factor this out such that we don't have to change these thresholds when the duration changes